Orange/tests/test_variable.py

```diff
@@ -247,6 +278,16 @@ def test_val(self):
         self.assertEqual(a.str_val(Value(a, "")), "?")
         self.assertEqual(a.repr_val(Value(a, "foo")), '"foo"')
 
+    def test_copy_copies_attributes(self):
+        var = StringVariable("x")
+        var.attributes["a"] = "b"
+        var2 = var.copy(compute_value=None)
+        self.assertIn("a", var2.attributes)
+
+        var2.attributes["a"] = "c"
+        # Attributes of original value should not change
+        self.assertEqual(var.attributes["a"], "b")
```

Comment:
I do not understand, which base method?

Time variable does not implement copy.